index.js

```diff
@@ -214,23 +214,24 @@ SteamTradeOffers.prototype.cancelOffer = function(options, callback) {
   });
 };
 
-SteamTradeOffers.prototype.acceptOffer = function(options, callback) {
+SteamTradeOffers.prototype.acceptOffer = function(offer, callback) {
```

Comment:
I see, but no one can guarantee this. Valve may add some property that needs to be computed or some other params may be added. Modifying `offer` object would look ugly.

Yes, please follow the convention.